pom.xml

```diff
@@ -50,7 +50,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <cassandra.version>3.11.2</cassandra.version>
-        <java.version>1.6</java.version>
+        <java.version>1.8</java.version>
```

Comment:
This is needed because Java 6 does not support static methods on interfaces, which are crucial for using OpenTelemetry API (Context, Attributes, etc.)